screens/SearchScreen.js

```diff
@@ -1,19 +1,16 @@
 import * as React from 'react';
 import { StyleSheet, Text, View } from 'react-native';
-import IndividualArticle from '../components/IndividualArticle'
 
 const SearchScreen = () => {
   return (
     <View style={styles.container}>
       <Text> Search Screen </Text>
-      <IndividualArticle/>
     </View>
-
   );
 };
 
-export default SearchScreen;
-
 const styles = StyleSheet.create({
   container:{}
 });
```

Comment:
What I meant was actually this line, as you don't use styles its better to remove it completely.

Reply:
Yea but we will add styles eventually and therefore need the stylesheet on the SearchScreen... Not sure if even I added that...